components/coda/coda.app.mjs

```diff
@@ -1,11 +1,158 @@
+import axios from "axios";
+
 export default {
   type: "app",
   app: "coda",
-  propDefinitions: {},
+  propDefinitions: {
+    title: {
+      type: "string",
+      label: "Doc Title",
+      description: "Title of the doc",
+      optional: true,
+    },
+    folderId: {
+      type: "string",
+      label: "Folder ID",
+      description: "The ID of the folder",
+      optional: true,
+    },
+    sourceDoc: {
+      type: "string",
+      label: "Source Doc ID",
+      description: "A doc ID from which to create a copy.",
+      optional: true,
+      async options () {
+        return this._getKeyValuePair(
+          (await this.listDocs()).items
+        );
+      },
+    },
+    isOwner: {
+      type: "boolean",
+      label: "Is Owner Docs",
+      description: "Show only docs owned by the user.",
+      optional: true,
+    },
+    isPublished: {
+      type: "boolean",
+      label: "Is Published Docs",
+      description: "Show only published docs.",
+      optional: true,
+      default: false,
+    },
+    query: {
+      type: "string",
+      label: "Search Query",
+      description: "Search term used to filter down results.",
+      optional: true,
+    },
+    isStarred: {
+      type: "boolean",
+      label: "Is Starred Docs",
+      description: "If true, returns docs that are starred. If false, returns docs that are not starred.",
+      optional: true,
+      default: false,
+    },
+    inGallery: {
+      type: "boolean",
+      label: "In Gallery Docs",
+      description: "Show only docs visible within the gallery.",
+      optional: true,
+      default: false,
+    },
+    workspaceId: {
+      type: "string",
+      label: "Workspace ID",
+      description: "Show only docs belonging to the given workspace.",
+      optional: true,
+    },
+    limit: {
+      type: "integer",
+      label: "Limit",
+      description: "Maximum number of results to return in this query.",
+      default: 25,
+      min: 1,
+      max: 50,
+    },
+    pageToken: {
+      type: "string",
+      label: "Page Token",
+      description: "An opaque token used to fetch the next page of results.",
+      optional: true,
+    },
+    paginate: {
+      type: "boolean",
+      label: "Auto-Paginate",
+      description: "By default, list all docs matching search results across all result pages. Set to `false` to limit results to the first page.",
+      optional: true,
+      default: true,
+    },
+  },
   methods: {
+    _getKeyValuePair(list) {
+      return list.map(
+        (e) => ({
+          label: e.name,
+          value: e.id,
+        })
+      );
+    },
     // this.$auth contains connected account data
     authKeys() {
```

Comment:
If you see these in app files, you can actually remove them. They're included in the app files we programmatically generate for new apps, but once we fill out the app file's code, we can remove the example.

Reply:
Issue addressed